upgrade/scripts/common/ncn-common.sh

```diff
@@ -30,6 +30,21 @@ trap 'err_report' ERR
 touch /etc/cray/upgrade/csm/myenv
 . /etc/cray/upgrade/csm/myenv
 
+if [[ -z ${CSM_RELEASE} ]]; then
+    echo "ERROR: CSM_RELEASE environment variable is not set and must be present in /etc/cray/upgrade/csm/myenv."
+    return 1
```

Comment:
If all 3 of these aren't set it would take 3 runs of the script to get all the missing env vars, I'm partial to doing like so:

```
rc=0

if ...
   rc=$((rc+1))
fi

repeat checks

if [ "${rc}" -gt 0 ]; then
return $rc
fi
```

To help expose errors to the user all at once instead of piecemeal.

Reply:
i like that -- let me change

Reply:
fixed -- thanks for the review!